postal/service.go

```diff
@@ -26,9 +28,9 @@ type Transport interface {
 
 //go:generate faux --interface MappingResolver --output fakes/mapping_resolver.go
 // MappingResolver serves as the interface that looks up platform binding provided
-// dependency mappings given a  SHA256 and a path to search for bindings
+// dependency mappings given a SHA256
 type MappingResolver interface {
-	FindDependencyMapping(SHA256, bindingPath string) (string, error)
+	FindDependencyMapping(SHA256 string, platformDir string) (string, error)
```

Comment:
@ryanmoran This has become the platform dir rather than full binding path. Technically it's a change in the API, but as you stated earlier, the only real implementation of this interface is internal.